adapters/Unity/UnityAdapter/GADMAdapterUnity.m

```diff
@@ -81,17 +81,25 @@ - (void)getInterstitial {
     [strongConnector adapter:self didFailAd:error];
     return;
   }
-  _isLoading = YES;
-  [[GADMAdapterUnitySingleton sharedInstance] requestInterstitialAdWithDelegate:self];
+
+  UnitySingletonCompletion completeBlock = ^(UnityAdsError *error, NSString *message) {
+    if(error) {
+      if (strongConnector) {
+        NSError *errorWithDescription = GADUnityErrorWithDescription(message);
+        [strongConnector adapter:self didFailAd:errorWithDescription];
+      }
+      return;
+    }
+    [UnityAds addDelegate:self];
```

Comment:
Even with the perPlacement load API, UnityAds can still only load 1 ad per placement at a time, correct? This change no longer goes through the singleton to make sure another ad wasn't already loaded for that placement.

My guess is for each callback, UnityAds invokes every registered delegate with that callback. If 2 interstitial adapters are listening in on the same placement, then 2 adapters will claim an ad has loaded when there's only 1 backing ad. That is why we previously went through the singleton class, to make sure that only 1 adapter per placement would stick around.